.travis.yml

```diff
@@ -1,19 +1,36 @@
 sudo: required
+
 services:
   - docker
+
 language: python
+
 python:
   - "3.6"
+
 install:
   - pip install pipenv
   - pipenv install --dev --deploy
   - pipenv check
   - pip install tox==2.7.0
+
 script:
   - pipenv run tox
+
 after_success:
-  - bash <(curl -s https://codecov.io/bash)
-notifications:
-    slack: surveydatacollection:LCbNJJpDoY3gYIvLOlK0nZkV
+- if ([ "$TRAVIS_BRANCH" == "rm-census" ] && [ "$TRAVIS_PULL_REQUEST" == "false" ]) || ([ "$TRAVIS_BRANCH" == "automate-census-docker-build" ]); then
+  docker build -t "$DESTINATION_IMAGE_NAME" .;
+  docker login -u "${DOCKER_GCP_USERNAME}" -p "${DOCKER_GCP_PASSWORD}" "${DOCKER_GCP_REGISTRY}";
+  docker push "$DESTINATION_IMAGE_NAME";
+  fi
+- bash <(curl -s https://codecov.io/bash)
+
 env:
-  - secure: ol0EDEa6z4g4DlbmGijpTrDf4OBaBW6OnD0YbhaD7tLGmW/HyxFlR8Wnhl2RZjNBXt+fw8yypVeNYn2enbiI5omWTdk110bZSWh97X1+0nFdf4UVBIXp8y36XTeQx9KiqwZC2mveX1z9CwI4/a+o3AOisUoUjiB5N9SbOteMHNckkZ2dHezfzBsbSNXB2kn+/tAYQCKqXrTGkpsMtdvO8LhVX7U7srqEqFAsBg2ZDY2Ry0SFLbm0QjwH+p31Y0oCvhNL8/XiYzpjiJZGtQzgWRnZ3vFbLmPg0zfgFqSqujiC6D9fZ81b3U7X6TJsOv7Hm6Ci2703MgSI9MaooRe/FE9AKowE9czoZiWVj3Uy8W9ojOJs3ipHJWDNuR4/Jnb/8Qny1tYFL7+nv+jGOJlaZwBkYsaQZGnra92tDHqkOBIm8dE67QCE39JqPVMN2BinWZnvIt0/CJfNL766QXiFV0oJbkVphFUnQQzJfOX6fwzyLRAaaHrqQ1flJVf1eLa4JG7YjRLSNySzJJ/x3ks9szTnOEiv9GbKInspDyEODckNAh6t9QVtNlQuKXh41Fhhcvrw+eOk7MbFOlRnk6pRGgNlzKWJycnHl2Yoft9fIlFO+4tCd3tL5zXxMc6SNV8x3vnlmYxkKn1idADML5dI8hXSudLnQakI+SkcvogaRTM=
+  global:
+    - secure: ol0EDEa6z4g4DlbmGijpTrDf4OBaBW6OnD0YbhaD7tLGmW/HyxFlR8Wnhl2RZjNBXt+fw8yypVeNYn2enbiI5omWTdk110bZSWh97X1+0nFdf4UVBIXp8y36XTeQx9KiqwZC2mveX1z9CwI4/a+o3AOisUoUjiB5N9SbOteMHNckkZ2dHezfzBsbSNXB2kn+/tAYQCKqXrTGkpsMtdvO8LhVX7U7srqEqFAsBg2ZDY2Ry0SFLbm0QjwH+p31Y0oCvhNL8/XiYzpjiJZGtQzgWRnZ3vFbLmPg0zfgFqSqujiC6D9fZ81b3U7X6TJsOv7Hm6Ci2703MgSI9MaooRe/FE9AKowE9czoZiWVj3Uy8W9ojOJs3ipHJWDNuR4/Jnb/8Qny1tYFL7+nv+jGOJlaZwBkYsaQZGnra92tDHqkOBIm8dE67QCE39JqPVMN2BinWZnvIt0/CJfNL766QXiFV0oJbkVphFUnQQzJfOX6fwzyLRAaaHrqQ1flJVf1eLa4JG7YjRLSNySzJJ/x3ks9szTnOEiv9GbKInspDyEODckNAh6t9QVtNlQuKXh41Fhhcvrw+eOk7MbFOlRnk6pRGgNlzKWJycnHl2Yoft9fIlFO+4tCd3tL5zXxMc6SNV8x3vnlmYxkKn1idADML5dI8hXSudLnQakI+SkcvogaRTM=
+    - PIPENV_IGNORE_VIRTUALENVS=1
+    - DESTINATION_IMAGE_NAME="$DOCKER_GCP_LOCATION/census-rm-collectioninstrumentsvc"
+
+branches:
+  only:
+    - automate-census-docker-build
```

Comment:
This could be set to the base branch so that it triggers PR builds rather than push builds (which are sometimes disabled in travis). Would reduce the amount that needs to be changed in between reviewing and merging the PR.